jupyter_client/kernelspecapp.py

```diff
@@ -33,6 +33,7 @@ def _kernel_spec_manager_default(self):
 
     def start(self):
         specs = self.kernel_spec_manager.find_kernel_specs()
+        specs = {kname:{"dir":specs[kname],"info":self.kernel_spec_manager.get_kernel_spec(kname).to_dict()} for kname in specs}
```

Comment:
We're not strict about keeping lines to 80 characters, but I think this is a bit too long. Can you split it onto two (or more) lines, and add a few spaces around the punctuation to make it  a bit more readable?